README.md

```diff
@@ -120,6 +120,39 @@ Useful if there is a bug in a native implementation, or a combination of a non-s
 
 Ex: Node 4 supports native classes but not spread.
 
+### `useBuiltIns`: `boolean`
+
+Defaults to `false`.
+
+A way to apply `babel-preset-env` for polyfills (via "babel-polyfill").
+
+```
+npm install babel-polyfill --save
+```
+
+This option will apply a new plugin that replaces the statement `import "babel-polyfill"` or `require("babel-polyfill")` with individual requires for `babel-polyfill` based on environment.
+
+> NOTE: This means you should only use `require("babel-polyfill");` once in your whole app. I would recommend just creating a single file that only does the require.
```

Comment:
Nit: I generally prefer not to use "I" in readmes unless there's a reason you want it to be explicit that you, the author, are saying something. Maybe something like, "Creating a single file that only does contains the require is recommended."?